### PR TITLE
Don't start browser server-side.

### PR DESCRIPTION
**Pull request by orhtej2**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by orhtej2** [comment]
!testme

File: conf/systemd.service [hunk 1]
```diff
@@ -6,7 +6,7 @@ Type=simple
 User=__APP__
 Group=__APP__
 WorkingDirectory=__INSTALL_DIR__/
-ExecStart=__INSTALL_DIR__/rclone rcd --rc-web-gui --rc-addr=127.0.0.1:__PORT__ --rc-user __ADMIN__ --rc-pass __PASSWORD__ --rc-web-gui-update
+ExecStart=__INSTALL_DIR__/rclone rcd --rc-web-gui --rc-addr=127.0.0.1:__PORT__ --rc-user __ADMIN__ --rc-pass __PASSWORD__ --rc-web-gui-update --rc-web-gui-no-open-browser
```
Comment (ericgaspar) [inline conf/systemd.service:9]:
```suggestion
ExecStart=__INSTALL_DIR__/rclone rcd --rc-web-gui-no-open-browser --rc-addr=127.0.0.1:__PORT__ --rc-user __ADMIN__ --rc-pass __PASSWORD__ --rc-web-gui-update
```

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
superseded by #10

**Pull request closed without merging**